basic/splunk/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+splunk:
+   host: localhost
+   port: 8089
+   username: admin
+   password: admin
+   owner: admin
```

Comment:
A new line in the end of each file would be good.
Minor, of course, but as you see this extra noise on review.
You can configure your IDE to add it for you.

I don't insist to do all that reconfiguration changes, BTW, but that will save our time on the next your contribution.